### ISSUE TITLE
New features Sign in and Sign up 

### ISSUE THREAD

**Issue by asaikarthikeya** [enviroCar/enviroCar-app#769]
1. Add option to add a profile pic
2. No option to recover if user forgets his password/ User name.
3. Currently, User doesnot have option to reset his password.
4. Gmail , Facebook and other Social accounts login.

**Comment by DhirajChauhan40** [comment]
Hey @asaikarthikeya I wanted to ask if the features have been implemented on the server-side?

**Comment by asaikarthikeya** [comment]
@DhirajChauhan40 No. This may take some more time. Thank you for showing your intrest ;-)

**Comment by cdhiraj40** [comment]
okay back to this, is this issue open for work? :)

**Comment by cdhiraj40** [comment]
@asaikarthikeya  okay back to this, is this issue open for work now? :)

**Comment by asaikarthikeya** [comment]
Not anytime in the near future. I will mention in the issue description if it is ready to be worked on.
